api/controllers/v1/UserController.coffee

```diff
@@ -0,0 +1,29 @@
+# UserController
+#
+# @description :: Server-side logic for managing users
+# @help        :: See http://sailsjs.org/#!/documentation/concepts/Controllers
+
+
+module.exports =
+
+  # `UserController.index()`
+
+  index: (req, res) ->
```

Comment:
If the tests are going to use `response` rather than `resp` should this not also be true of the actual controller code?